build-container.ps1

```diff
@@ -11,10 +11,18 @@ $BaseTable = @{
     2004 = "mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-2004";
     ## 20H2 reports itself as 2009 in the registry
     2009 = "mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-20H2"
+    2022 = "mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2022"
 }
 
 $kernelver = [int](get-itemproperty -path "hklm:software\microsoft\windows nt\currentversion" -name releaseid).releaseid
```

Comment:
I've changed it to use the build number. I'm comparing it with "greater or equal" and not just equal because I assume it will go up on future revisions.